ci/refresh-get-daml-com.yml

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+jobs:
+- job: get-daml-com
+  condition: eq(variables['Build.SourceBranchName'], 'main')
+  pool:
+    name: 'ubuntu_20_04'
+    demands: assignment -equals default
+  steps:
+    - checkout: self
+    - template: ci/bash-lib.yml
+      parameters:
+        var_name: bash-lib
+    - bash: |
+        set -euo pipefail
+        source $(bash-lib)
+        tmp=$(mktemp)
+        curl --silent \
+             --fail \
+             --location \
+             https://get.daml.com \
+         > $tmp
+        if diff daml-assistant/get-daml.sh $tmp; then
+          echo "Files match, nothing to do."
+        else
+          echo "Files differ; pushing new version."
+          aws s3 cp daml-assistant/get-daml.sh s3://get-daml-com/get-daml.sh --acl public-read --region us-east-1
+          echo "Telling Slack..."
+          tell_slack "Change detected in get.daml.com:\`\`\`$(diff daml-assistant/get-daml.sh $tmp)\`\`\`"
```

Comment:
I imagine that doc changes might often be largish? In which case, would a `diff --stat` (to get a summary diff) be better here?

Reply:
Oh, just a moment, we're only looking at `get-dams.sh` changing here - so scrub this comment!

Reply:
It's just that one file, and I don't expect it to change much, or often. There's an argument to be made for better display (e.g. put the diff in a Slack thread), but I don't think the extra work is worth it in this case.